### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Created Static Homepage
Add Navbar & SearchBar
Render the data on the Page
vl-1

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,13 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import { Routes, Route, Link } from "react-router-dom";
+import {Navbar} from "./components/index"
+import {Searchbar} from "./components/index"
```
Comment (therajatg) [inline src/App.js:5]: could import Navbar and searchbar at once

File: src/components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    --white-color: #ffffff;
+    --primary-color: #AB542F;
+}
+
+
+.navbar {
+display: flex;
+justify-content:space-between;
+height: 9vh;
+/* align-items: center; */
```
Comment (therajatg) [inline src/components/navbar/navbar.css:13]: remove comment if u don't need it

**Review by therajatg — APPROVED** [review]

**Review by harshitpaliwal95 — COMMENTED** [review]
- file structure is good
- code is clean
- good to go you can merge

File: src/App.js (hunk 1 above)
Comment (agrawalshanu708) [reply src/App.js:5]: ohkay rajat thanks

File: src/components/navbar/navbar.css (hunk 2 above)
Comment (agrawalshanu708) [reply src/components/navbar/navbar.css:13]: ohkay

**Review by harshitpaliwal95 — APPROVED** [review]

File: src/components/videostore/videostore.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.videostore{
```
Comment (tanishk15sharma) [inline src/components/videostore/videostore.css:1]: it no style is given then remove this

**Comment by yogesh-** [comment]
- file structure is good
- code is modular
Can be merged.

**Pull request merged**